sandbox/indeed-scraping/indeed-scraper.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+
+import sys
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.common.exceptions import TimeoutException
+import time
+import requests
+import random
+import pandas as pd
+import boto3
+import botocore
+import json
+import logging
```

Comment:
I'm glad you chose to use logging, nice

Reply:
yeah, we can check the log file indeed-scraper-<search-string>.log for details